yearn/utils.py

```diff
@@ -117,9 +122,14 @@ def __call__(self, *args, **kwargs):
 # cached Contract instance, saves about 20ms of init time
 _contract_lock = threading.Lock()
 _contract = lru_cache(maxsize=None)(Contract)
+_interface = lru_cache(maxsize=None)(interface.ERC20)
```

Comment:
this isn't necessarily an ERC20 though right?

Reply:
that's right, I've changed the mapping so we can define per contract which interface should be used.